py17track/profile.py

```diff
@@ -78,6 +81,7 @@ async def packages(
                 "info_text": event.get("z"),
                 "location": event.get("c"),
                 "timestamp": event.get("a"),
+                "tz": tz,
```

Comment:
What does this do for us? We're setting the timezone based on a kwarg?

Reply:
I have no experience with attrs and wouldn't know how to get this parameter over to the post_init method.

Reply:
Ah, apologies; I see what you're doing. This is fine.